akka-docs-dev/rst/scala/code/docs/http/scaladsl/server/WebsocketExampleSpec.scala

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (C) 2009-2015 Typesafe Inc. <http://www.typesafe.com>
+ */
+
+package docs.http.scaladsl.server
+
+import org.scalatest.{ Matchers, WordSpec }
+
+class WebsocketExampleSpec extends WordSpec with Matchers {
+  "core-example" in {
+    pending // compile-time only test
+    import akka.actor.ActorSystem
+    import akka.stream.ActorFlowMaterializer
+    import akka.stream.scaladsl.{ Source, Flow }
+
+    import akka.http.scaladsl.Http
+    import akka.http.scaladsl.model.HttpMethods._
+    import akka.http.scaladsl.model.{ HttpResponse, Uri, HttpRequest }
+    import akka.http.scaladsl.model.ws.UpgradeToWebsocket
+    import akka.http.scaladsl.model.ws.{ TextMessage, Message }
```

Comment:
Include the imports in the code snippet? IDEs should autocomplete them but it helps a lot for newbies to see the imports in docs.